src/core/query.spec.js

```diff
@@ -934,6 +934,20 @@ describe('Query', () => {
       expect(result[0].customProperty).to.equal(entity1.customProperty);
       expect(result[1].customProperty).to.equal(entity2.customProperty);
     });
+
+    it('should put docs with null values for sort field at the beginning of the list', () => {
+      const entity1 = { _id: 1, customProperty: randomString() };
+      const entity2 = { _id: null, customProperty: randomString() };
+      const entity3 = { _id: 2, customProperty: randomString() };
+      const entity4 = { _id: 3, customProperty: randomString() };
+      const query = new Query().ascending('_id');
+      query.fields = ['customProperty'];
+      const result = query.process([entity4, entity1, entity3, entity2]);
```

Comment:
It would help to define the spec more thoroughly to add an entity which does not contain a `someProperty` field at all. Or maybe an object, which is a primitive even - are those treated the same as if they are objects, which don't have it?